src/Microsoft.Diagnostics.Monitoring/Egress/EgressProvider.cs

```diff
@@ -0,0 +1,114 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
This folder is where I meant to comment, can we move the functionality here to the REST server?